### PR TITLE
Fix development environment setup

### PR DESCRIPTION
**Pull request by ilcsik**
Fixes for setting up development environment.

### CODE REVIEW

File: dev_environ_setup.md [hunk 1]
```diff
@@ -33,9 +43,16 @@ Step 4. Create Postgres user and PostGIS required extensions:
...
 # CREATE EXTENSION postgis_topology;
 # CREATE EXTENSION fuzzystrmatch;
+# CREATE EXTENSION hstore;
 # \q
 ```
 
+Modifiy the base template so all newly created test databases will have hstore extension:
+
+```bash
+$ psql -d template1 -c 'CREATE EXTENSION hstore;'
```
Comment (robertavram) [inline dev_environ_setup.md]: :+1:

File: EquiTrack/requirements/base.txt [hunk 2]
```diff
@@ -52,6 +52,7 @@ django-easy-pdf
 xhtml2pdf==0.0.6
 html5lib==0.9999999 #must be added after xhtml2pdf
 reportlab
+sqlparse==0.1.19  # for django debug toolbar to not break admin
```
Comment (robertavram) [inline EquiTrack/requirements/base.txt]: let's create another file for requirements called local.txt and add `-r base.txt` to it
add requirements that are only needed locally to that file.

Comment (ilcsik) [inline EquiTrack/requirements/base.txt]: Done in https://github.com/unicef/etools/pull/81/commits/4df8c39789d07f0d7773a4fbc4f2854b1965e50b

**Comment by fafcrumb** [comment]
Should I reflect these changes in the linux version?

**Comment by ilcsik** [comment]
I don't have a linux box to try, do you? Most changes should apply, except postgres install of course.

**Comment by robertavram** [comment]
can you pull this against Develop branch instead of Master

**Pull request closed without merging**